rpcs3@0.0.37-18014: download via aria2 failed

```
rpcs3: 0.0.37-18009 -> 0.0.37-18014
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
WARN  To disable this warning, run 'scoop config aria2-warning-enabled false'.
Updating one outdated app:
Updating 'rpcs3' (0.0.37-18009 -> 0.0.37-18014)
Downloading new version
Starting download with aria2 ...
Download: 06/18 08:44:15 [ERROR] CUID#8 - Download aborted. URI=https://github.com/rpcs3/rpcs3-binaries-win/releases/dowDownload: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://github.com/rpcs3/rpcs3-binaries-win/releases/downlDownload: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: ea4c25|ERR |       0B/s|C:/Users/user/scoop/cache/rpcs3#0.0.37-18014#a2980b5.7z
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://github.com/rpcs3/rpcs3-binaries-win/releases/download/build-27359d3d329693c3f6bab2d480a08603511f2261/rpcs3-v0.0.37-18014-27359d3d_win64.7z
    referer=https://github.com/rpcs3/rpcs3-binaries-win/releases/download/build-27359d3d329693c3f6bab2d480a08603511f2261/
    dir=C:\Users\user\scoop\cache
    out=rpcs3#0.0.37-18014#a2980b5.7z

ERROR & 'C:\Users\user\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\user\scoop\cache\rpcs3.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.5 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=40108 --continue --summary-interval=0 --auto-save-interval=1 --disable-ipv6

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/Calinou/scoop-games/issues/new?title=rpcs3%400.0.37-18014%3a+download+via+aria2+failed
```

--- a/bucket/rpcs3.json
+++ b/bucket/rpcs3.json
@@ -15,7 +15,7 @@
     "suggest": {
         "Microsoft Visual C++ Runtime 2022": "extras/vcredist2022"
     },
-    "url": "https://github.com/rpcs3/rpcs3-binaries-win/releases/download/build-329655a1bff2460bdd388397184a1bcb18373d76/rpcs3-v0.0.37-18025-329655a1_win64.7z",
+    "url": "https://github.com/RPCS3/rpcs3-binaries-win/releases/download/build-329655a1bff2460bdd388397184a1bcb18373d76/rpcs3-v0.0.37-18025-329655a1_win64_msvc.7z",
     "hash": "237637871565723b345ef709ee23b11e44c35effa53f39c02344746811c8f05d",
     "pre_install": [
         "if (!(Test-Path \"$persist_dir\")) {",
@@ -50,15 +50,14 @@
         "patches"
     ],
     "checkver": {
-        "url": "https://rpcs3.net/compatibility?b",
-        "regex": "/rpcs3-binaries-win/releases/download/build-(?<fullhash>[0-9a-f]+)/rpcs3-v(?<build>[0-9]+\\.[0-9]+\\.[0-9]+\\-[0-9]+)-(?<shorthash>[0-9a-f]{8})",
-        "replace": "${build}"
+        "url": "https://api.github.com/repos/RPCS3/rpcs3-binaries-win/releases/latest",
+        "jsonpath": "$.assets[?(@.name =~ /rpcs3-v[a-f\\d.-]+_win64_msvc\\.7z/)].browser_download_url",
+        "regex": "releases/download/(?<build>build-[a-f\\d]+)/rpcs3-v(?<version>[\\d.-]+)-(?<commit>[a-f\\d]+)_win64_msvc\\.7z"
     },
     "autoupdate": {
-        "url": "https://github.com/rpcs3/rpcs3-binaries-win/releases/download/build-$matchFullhash/rpcs3-v$matchBuild-$matchShorthash_win64.7z",
+        "url": "https://github.com/RPCS3/rpcs3-binaries-win/releases/download/$matchBuild/rpcs3-v$version-$matchCommit_win64_msvc.7z",
         "hash": {
-            "url": "https://rpcs3.net/compatibility?b",
-            "regex": "Windows x64 SHA-256: ($sha256)"
+            "url": "https://github.com/RPCS3/rpcs3-binaries-win/releases/download/$matchBuild/rpcs3-v$version-$matchCommit_win64_msvc.7z.sha256"
         }
     }
 }

